pkg/controllers/disruption/orchestration/metrics.go

```diff
@@ -24,7 +24,7 @@ import (
 )
 
 func init() {
```

Comment:
There is more than just a single place where we create the workqueue. Should this go in the disruption orchestration package or should this be in the top-level metrics package? I believe this workqueue is also going to apply to the eviction queue that we create

Reply:
Agreed. Moved it to the top level metrics package.